compose/server/Dockerfile

```diff
@@ -9,5 +9,5 @@ RUN pip install --upgrade pip ipdb ipython
 
 COPY . /usr/src/app/
 
-RUN pip install bigchaindb
+RUN pip install git+https://github.com/bigchaindb/bigchaindb.git
```

Comment:
I didn't want to change this, but if we'll change this back later then it's fine.